math/pi/montecarlopi.go

```diff
@@ -0,0 +1,27 @@
+// montecarlopi.go
+// description: Calculating pi by the Monte Carlo method
+// details:
+// implementation of Monte Carlo Algorithm for the calculating of Pi - [Monte Carlo method](https://en.wikipedia.org/wiki/Monte_Carlo_method)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see montecarlopi_test.go
+
+package pi
+
+import (
+	"math/rand"
+	"time"
+)
+
+func MonteCarloPi(n int) float64 {
+	rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Good - when I looked at it in the morning this was not clear. I didn't have time to write the comments but I like this change. 😄